github-management/README.md

```diff
@@ -19,6 +19,27 @@ Experience Special Interest Group.
 - [Setting up the CNCF CLA Check](setting-up-cla-check.md)
 - [GitHub Permissions](permissions.md)
 
+## GitHub Administration Team
+
+In order to manage the various organizations that the Kubernetes project owns,
+we have a GitHub Administration team that is responsible for carrying out the
+various tasks.
+
+This team (**[@kubernetes/owners](https://github.com/orgs/kubernetes/teams/owners)**) is as follows:
+* Aaron Crickenberger (**[@spiffxp](https://github.com/spiffxp)**, Pacific Time)
+* Christoph Blecker (**[@cblecker](https://github.com/cblecker)**, Pacific Time)
+* Erick Fejta (**[@fejta](https://github.com/fejta)**, Pacific Time)
+* Garrett Rodrigues (**[@grodrigues3](https://github.com/grodrigues3)**, Pacific Time)
+* Ihor Dvoretskyi (**[@idvoretskyi](https://github.com/idvoretskyi)**, Eastern European Time)
```

Comment:
I think it would be good to mention that we need at least one owner in some European time zone and outside the US to increase holiday coverage.

Reply:
Added